<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KForecastViewDecl</class>
  <widget class="QWidget" name="KForecastViewDecl">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>909</width>
        <height>771</height>
      </rect>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>0</number>
      </property>
      <item>
        <widget class="Q3GroupBox" name="groupBox3">
          <property name="title">
            <string>Forecast Settings</string>
          </property>
          <layout class="QHBoxLayout">
            <item>
              <layout class="QGridLayout">
                <item row="0" column="0">
                  <widget class="QLabel" name="textLabel1">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>5</hsizetype>
                        <vsizetype>5</vsizetype>
                        <horstretch>1</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text">
                      <string>Days to Forecast:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1">
                  <widget class="QSpinBox" name="m_forecastDays">
                    <property name="maximum">
                      <number>999</number>
                    </property>
                    <property name="minimum">
                      <number>1</number>
                    </property>
                  </widget>
                </item>
                <item row="1" column="0">
                  <widget class="QLabel" name="textLabel3">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>5</hsizetype>
                        <vsizetype>5</vsizetype>
                        <horstretch>1</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text">
                      <string>Days of Accounts Cycle:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="1" column="1">
                  <widget class="QSpinBox" name="m_accountsCycle">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>1</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="maximum">
                      <number>999</number>
                    </property>
                    <property name="minimum">
                      <number>1</number>
                    </property>
                  </widget>
                </item>
                <item row="2" column="0">
                  <widget class="QLabel" name="textLabel5">
                    <property name="text">
                      <string>Day of Month to start Forecast:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="2" column="1">
                  <widget class="QSpinBox" name="m_beginDay">
                    <property name="maximum">
                      <number>31</number>
                    </property>
                    <property name="minimum">
                      <number>0</number>
                    </property>
                  </widget>
                </item>
                <item row="3" column="0">
                  <widget class="QLabel" name="textLabel4">
                    <property name="text">
                      <string>Historic Cycles:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="3" column="1">
                  <widget class="QSpinBox" name="m_forecastCycles">
                    <property name="maximum">
                      <number>999</number>
                    </property>
                    <property name="minimum">
                      <number>1</number>
                    </property>
                  </widget>
                </item>
                <item row="4" column="0">
                  <widget class="QLabel" name="textLabel6">
                    <property name="text">
                      <string>Chart Detail:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="4" column="1">
                  <widget class="KComboBox" name="m_comboDetail">
                    <property name="currentIndex">
                      <number>2</number>
                    </property>
                    <property name="toolTip" stdset="0">
                      <string>&lt;p&gt;Choose what level of detail to show on the chart.&lt;/p&gt;</string>
                    </property>
                    <item>
                      <property name="text">
                        <string comment="Show all details">All</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Top-Level</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Groups</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Totals</string>
                      </property>
                    </item>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <spacer name="spacer23">
                <property name="sizeHint">
                  <size>
                    <width>16</width>
                    <height>31</height>
                  </size>
                </property>
                <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
              </spacer>
            </item>
            <item>
              <layout class="QVBoxLayout">
                <item>
                  <layout class="QVBoxLayout">
                    <item>
                      <layout class="QHBoxLayout">
                        <item>
                          <widget class="QLabel" name="textLabel2">
                            <property name="sizePolicy">
                              <sizepolicy>
                                <hsizetype>5</hsizetype>
                                <vsizetype>5</vsizetype>
                                <horstretch>1</horstretch>
                                <verstretch>0</verstretch>
                              </sizepolicy>
                            </property>
                            <property name="text">
                              <string>Current Forecast Method:</string>
                            </property>
                            <property name="wordWrap">
                              <bool>false</bool>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget class="QLabel" name="m_forecastMethod">
                            <property name="sizePolicy">
                              <sizepolicy>
                                <hsizetype>5</hsizetype>
                                <vsizetype>5</vsizetype>
                                <horstretch>2</horstretch>
                                <verstretch>0</verstretch>
                              </sizepolicy>
                            </property>
                            <property name="font">
                              <font>
                                <bold>true</bold>
                              </font>
                            </property>
                            <property name="text">
                              <string>method</string>
                            </property>
                            <property name="wordWrap">
                              <bool>false</bool>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </item>
                    <item row="2" column="0">
                      <widget class="KButtonGroup" name="m_historyMethod">
                        <property name="title">
                          <string>History Forecast Method</string>
                        </property>
                        <layout class="QVBoxLayout">
                          <item>
                            <widget class="QRadioButton" name="radioButton11">
                              <property name="text">
                                <string>Simple Moving Average</string>
                              </property>
                            </widget>
                          </item>
                          <item>
                            <widget class="QRadioButton" name="radioButton12">
                              <property name="text">
                                <string>Weighted Moving Average</string>
                              </property>
                            </widget>
                          </item>
                          <item>
                            <widget class="QRadioButton" name="radioButton12">
                              <property name="text">
                                <string>Linear Regression</string>
                              </property>
                            </widget>
                          </item>
                        </layout>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <spacer name="spacer7">
                    <property name="sizeHint">
                      <size>
                        <width>21</width>
                        <height>6</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Vertical</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <layout class="QHBoxLayout">
                    <item>
                      <widget class="KPushButton" name="m_forecastButton">
                        <property name="text">
                          <string>Forecast</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <spacer name="spacer22">
                        <property name="sizeHint">
                          <size>
                            <width>31</width>
                            <height>21</height>
                          </size>
                        </property>
                        <property name="sizeType">
                          <enum>QSizePolicy::Expanding</enum>
                        </property>
                        <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                        </property>
                      </spacer>
                    </item>
                    <item>
                      <widget class="KPushButton" name="m_saveButton">
                        <property name="enabled">
                          <bool>false</bool>
                        </property>
                        <property name="text">
                          <string>Save</string>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="KTabWidget" name="m_tab">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>7</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
            </sizepolicy>
          </property>
          <widget class="QWidget" name="summary">
            <attribute name="title">
              <string>Summary</string>
            </attribute>
            <layout class="QVBoxLayout">
              <item>
                <layout class="QHBoxLayout">
                  <item>
                    <widget class="KMyMoneyAccountTreeForecast" name="m_summaryList">
                      <property name="sizePolicy">
                        <sizepolicy>
                          <hsizetype>7</hsizetype>
                          <vsizetype>7</vsizetype>
                          <horstretch>2</horstretch>
                          <verstretch>1</verstretch>
                        </sizepolicy>
                      </property>
                      <property name="midLineWidth">
                        <number>1</number>
                      </property>
                      <property name="resizePolicy">
                        <enum>AutoOneFit</enum>
                      </property>
                      <column>
                        <property name="text">
                          <string>Account</string>
                        </property>
                        <property name="clickable">
                          <bool>true</bool>
                        </property>
                        <property name="resizable">
                          <bool>true</bool>
                        </property>
                      </column>
                      <column>
                        <property name="text">
                          <string comment="0 days from now">0 days</string>
                        </property>
                        <property name="clickable">
                          <bool>true</bool>
                        </property>
                        <property name="resizable">
                          <bool>true</bool>
                        </property>
                      </column>
                      <column>
                        <property name="text">
                          <string comment="30 days from now">30 days</string>
                        </property>
                        <property name="clickable">
                          <bool>true</bool>
                        </property>
                        <property name="resizable">
                          <bool>true</bool>
                        </property>
                      </column>
                      <column>
                        <property name="text">
                          <string comment="60 days from now">60 days</string>
                        </property>
                        <property name="clickable">
                          <bool>true</bool>
                        </property>
                        <property name="resizable">
                          <bool>true</bool>
                        </property>
                      </column>
                      <column>
                        <property name="text">
                          <string comment="90 days from now">90 days</string>
                        </property>
                        <property name="clickable">
                          <bool>true</bool>
                        </property>
                        <property name="resizable">
                          <bool>true</bool>
                        </property>
                      </column>
                    </widget>
                  </item>
                  <item>
                    <widget class="Q3TextEdit" name="m_adviceText">
                      <property name="sizePolicy">
                        <sizepolicy>
                          <hsizetype>7</hsizetype>
                          <vsizetype>7</vsizetype>
                          <horstretch>1</horstretch>
                          <verstretch>1</verstretch>
                        </sizepolicy>
                      </property>
                      <property name="resizePolicy">
                        <enum>Q3ScrollView::AutoOneFit</enum>
                      </property>
                      <item>
                        <property name="text">
                          <string>New Item</string>
                        </property>
                        <property name="pixmap">
                          <pixmap/>
                        </property>
                      </item>
                    </widget>
                  </item>
                </layout>
              </item>
              <item>
                <layout class="QHBoxLayout"/>
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="detail">
            <attribute name="title">
              <string>Detail</string>
            </attribute>
            <layout class="QVBoxLayout">
              <property name="margin">
                <number>0</number>
              </property>
              <item>
                <widget class="KMyMoneyAccountTreeForecast" name="m_forecastList">
                  <property name="resizePolicy">
                    <enum>AutoOneFit</enum>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="advanced">
            <attribute name="title">
              <string comment="Advanced forecast view">Advanced</string>
            </attribute>
            <layout class="QVBoxLayout">
              <property name="margin">
                <number>0</number>
              </property>
              <item>
                <widget class="K3ListView" name="m_advancedList">
                  <property name="resizePolicy">
                    <enum>AutoOneFit</enum>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="budget">
            <attribute name="title">
              <string>Budget Forecast</string>
            </attribute>
            <layout class="QVBoxLayout">
              <property name="margin">
                <number>0</number>
              </property>
              <item>
                <widget class="KMyMoneyAccountTreeForecast" name="m_budgetList">
                  <property name="resizePolicy">
                    <enum>AutoOneFit</enum>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="m_tabChart">
            <attribute name="title">
              <string>Chart</string>
            </attribute>
            <layout class="QVBoxLayout">
              <property name="margin">
                <number>0</number>
              </property>
            </layout>
          </widget>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
    <customwidgets>
    <customwidget>
      <class>KMyMoneyAccountTreeforecast</class>
      <extends>QWidget</extends>
      <header location="local">../widgets/kmymoneyaccounttreeforecast.h</header>
      <sizehint>
        <width>-1</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
    </customwidget>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
      <header location="global">k3listview.h</header>
    </customwidget>
    <customwidget>
      <class>KTabWidget</class>
      <extends>QTabWidget</extends>
      <header location="global">ktabwidget.h</header>
    </customwidget>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
      <header location="global">kcombobox.h</header>
    </customwidget>
  </customwidgets>
</ui>
